drivers/usb/device/usb_dc_stm32.c

```diff
@@ -246,7 +290,7 @@ static int usb_dc_stm32_clock_enable(void)
 
 	/* Set the OTG PHY reference clock selection (through SYSCFG) block */
 	LL_APB3_GRP1_EnableClock(LL_APB3_GRP1_PERIPH_SYSCFG);
-	HAL_SYSCFG_SetOTGPHYReferenceClockSelection(SYSCFG_OTG_HS_PHY_CLK_SELECT_1);
+	HAL_SYSCFG_SetOTGPHYReferenceClockSelection(USB_PHY_HSE_CLOCK_SPEED);
```

Comment:
Macro name should be changed because the reference clock can also come from the PLL:
```suggestion
	HAL_SYSCFG_SetOTGPHYReferenceClockSelection(OTG_HS_PHY_REFERENCE_CLOCK);
```

Reply:
Ah yeah, it looks like it can. I just re-checked the STM32U5 clock diagram.

For reference, RM0456 rev 5, Figure 36 (Clock tree for STM32U5 Series)

![image](https://github.com/user-attachments/assets/c6d85b20-accd-4cb9-b7fc-c8ef412f72d0)

Will go rename this, thanks!

Reply:
Fixed!